crates/holochain/src/core/state/cascade.rs

```diff
@@ -58,20 +55,106 @@ mod test;
 
 pub mod error;
 
-pub struct Cascade<'a, Network = HolochainP2pCell, MetaVault = MetadataBuf, MetaCache = MetadataBuf>
+/////////////////
+// Helper macros
+/////////////////
+
+/// Get an item from an option
+/// or return early from the function
+macro_rules! get_or_return {
```

Comment:
```suggestion
macro_rules! ok_or_return {
```

Would be nice to not use `get` since that has a highly specialized meaning for us. This would match `ok_or_else`